streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/master/graphmanager/GraphManager.java

```diff
@@ -0,0 +1,47 @@
+package org.ray.streaming.runtime.master.graphmanager;
+
+import java.io.Serializable;
+import org.ray.streaming.jobgraph.JobGraph;
+import org.ray.streaming.runtime.core.graph.Graphs;
+import org.ray.streaming.runtime.core.graph.executiongraph.ExecutionGraph;
+
+/**
+ * The central role of graph management in JobMaster.
+ */
+public interface GraphManager extends Serializable {
+
+  /**
+   * Build execution graph from job graph.
+   * @param jobGraph logical plan of streaming job
+   * @return physical plan of streaming job
+   */
+  ExecutionGraph buildExecutionGraph(JobGraph jobGraph);
+
+  /**
+   * Set up execution vertex.
+   * @param jobGraph logical plan of streaming job
+   * @return physical plan of streaming job
+   */
+  ExecutionGraph setupExecutionVertex(JobGraph jobGraph);
```

Comment:
Why we can't build graph in `setupExecutionVertex`, and do this in tow step?

Reply:
It is one step. Change method name to 'setupStructure' for better understanding.